py/desida/fiberassign.py

```diff
@@ -0,0 +1,143 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst.
+"""
+==================
+desida.fiberassign
+==================
```

Comment:
I suggest that we name this something like `archive_fiberassign` to distinguish it from actual fiberassignment.